src/main/resources/templates/progress/list.html

```diff
@@ -1,46 +1,39 @@
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700&amp;subset=latin-ext" rel="stylesheet">
+    <link rel="stylesheet" th:href="@{/css/style.css}"/>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+    <title>Progres</title>
+</head>
 <body>
 <div class="container">
-    <h2>Šifry</h2>
-    <div class="table-responsive">
-        <table class="table" border="1">
-            <thead>
-            <tr>
-                <th>Jméno</th>
-                <th>Pořadí</th>
-                <th>Souřadnice</th>
-                <th>Akce</th>
-            </tr>
-            <tr th:if="${cyphers.empty}">
-                <td colspan="4"><p> V systému nejsou šifry</p></td>
-            </tr>
-            </thead>
-            <tbody>
-            <tr th:each="cypher : ${cyphers}">
-                <td th:text="${cypher.name}" align="center">Jmeno</td>
-                <td th:text="${cypher.stage}" align="center">Cislo</td>
-                <td th:text="${cypher.location}">X, Y</td>
-                <td align="center">
-                    <a href="#"
-                       class="btn btn-default btn-lg" th:href="@{/game/progress/stage(cypherId=${cypher.cypherId})}">
-                        Stav
-                    </a>
-                </td>
-            </tr>
-            </tbody>
-        </table>
+    <div class="row">
+        <div class="col-sm-12">
+            <h1>Šifry</h1>
+            <div th:if="${cyphers.empty}">
+                <p>V systému nejsou šifry<p/>
+            </div>
+
+            <div th:each="cypher : ${cyphers}" class="cypher organizer">
+                <a th:href="@{/game/progress/stage(cypherId=${cypher.cypherId})}">
+                    <div class="row align-items-center">
+                        <div th:text="|${cypher.stage}. ${cypher.name}|" class="col"></div>
+                    </div>
+                </a>
+            </div>
+        </div>
     </div>
+
     <form action="/logout" method="post">
         <!--<input type="hidden" th:name="${_csrf.parameterName}" th:value="${_csrf.token}"/>-->
```

Comment:
Tento komentář souvisí nějak s tím logoutem?

Reply:
ten komentar je starty (jeste z doby kdyz se delala security pro lusteniny :))
mame csrf vypnute, takze neoverujeme token -> proto je to zakomentovane